kitsune/search/v2/documents.py

```diff
@@ -67,18 +67,19 @@ def prepare_topic_ids(self, instance):
     def prepare_product_ids(self, instance):
         return [product.id for product in instance.products.all()]
 
-    def prepare_parent_id(self, instance):
-        if instance.parent:
-            return instance.parent.id
-        return None
-
     def prepare_display_order(self, instance):
         return instance.original.display_order
 
     @classmethod
     def get_model(cls):
         return wiki_models.Document
 
+    @classmethod
+    def get_queryset(cls):
+        return wiki_models.Document.objects.select_related(
```

Comment:
comments/docstrings pls

Reply:
Actually I'm going to move this into a separate PR, as I'm pretty sure we can remove the calls to `parent` since we're updating stuff, which will require some further discussion - anyhow, it's not related to signals, and only here because I wanted to quickly optimise it a bit for when I was doing some tests with a fully indexed elastic instance.